packages/commonwealth/server/migrations/20241022173454-token-modification.js

```diff
@@ -0,0 +1,257 @@
+          { name: 'totalSupply', type: 'uint256', internalType: 'uint256' },
+          { name: 'curveId', type: 'uint256', internalType: 'uint256' },
+          { name: 'scalar', type: 'uint256', internalType: 'uint256' },
+          { name: 'lphook', type: 'address', internalType: 'address' },
+          { name: 'launchAction', type: 'address', internalType: 'address' },
+        ],
+        outputs: [],
+        stateMutability: 'payable',
+      },
+      {
+        type: 'function',
+        name: 'protocolFee',
+        inputs: [],
+        outputs: [{ name: '', type: 'uint256', internalType: 'uint256' }],
+        stateMutability: 'view',
+      },
+      {
+        type: 'function',
+        name: 'protocolVault',
+        inputs: [],
+        outputs: [{ name: '', type: 'address', internalType: 'address' }],
+        stateMutability: 'view',
+      },
+      {
+        type: 'event',
+        name: 'LaunchpadCreated',
+        inputs: [
+          {
+            name: 'launchpad',
+            type: 'address',
+            indexed: false,
+            internalType: 'address',
+          },
+        ],
+        anonymous: false,
+      },
+      {
+        type: 'event',
+        name: 'NewTokenCreated',
+        inputs: [
+          {
+            name: 'token',
+            type: 'address',
+            indexed: false,
+            internalType: 'address',
+          },
+          {
+            name: 'totalSupply',
+            type: 'uint256',
+            indexed: false,
+            internalType: 'uint256',
+          },
+          {
+            name: 'name',
+            type: 'string',
+            indexed: false,
+            internalType: 'string',
+          },
+          {
+            name: 'symbol',
+            type: 'string',
+            indexed: false,
+            internalType: 'string',
+          },
+        ],
+        anonymous: false,
+      },
+      {
+        type: 'event',
+        name: 'TokenRegistered',
+        inputs: [
+          {
+            name: 'token',
+            type: 'address',
+            indexed: false,
+            internalType: 'address',
+          },
+          {
+            name: 'curveId',
+            type: 'uint256',
+            indexed: false,
+            internalType: 'uint256',
+          },
+        ],
+        anonymous: false,
+      },
+    ];
+    const hashInstance = hasher({
+      coerce: true,
+      sort: true,
+      trim: true,
+      alg: 'sha256',
+      enc: 'hex',
+    });
+    const launchpadHash = hashInstance.hash(namespaceAbi);
+
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.dropTable('Tokens', { transaction: t });
+      await queryInterface.changeColumn(
+        'Communities',
+        'namespace',
+        {
+          type: Sequelize.STRING,
+          allowNull: true,
+          unique: true,
+        },
+        { transaction: t },
+      );
+      await queryInterface.createTable(
```

Comment:
Why drop and recreate the table?

Reply:
We modify/add/remove many fields from the old model. And the old model is not in use on prod currently, so I think it is best to just recreate it

Reply:
Won't this affect testing for any already deployed token communities on QA/demo? Just want to make sure were not dropping tokens Sachin/Malik/Marcin expect to exist. Feel free to resolve otherwise.

Reply:
resolved by making it an update command instead of the drop/create